Fix for Radembedded tests failures

aravindprasads opened the pull request:
Description:
--
This is the fix for the issue https://github.com/radcli/radcli/issues/52. 
Currently, the tests for radembedded.c and radembedded_dict.c are failing. 

Root-Cause:
--
Currently, the Server IPs are statically assigned in radembedded files. Hence, they are not able to reach  the IP Address of dynamically spawned RADIUS Server using Docker.

Fix:
--
The IP configs in radembedded files must be made dynamic based on Radius Server IP of Docker. 
The Radembedded files are made to take the IP Address as command line argument and use it for their configs.

aravindprasads committed:
Fix for Radembeddded Test failures

aravindprasads commented:
Hi @nmav 

Kindly review the changes and let me know your views.

nmav commented on src/radembedded.c:
```diff
@@ -74,14 +83,30 @@ main (int argc, char **argv)
+	 */
+	if(argc == 2)
+	{
+		strncpy(auth_server_ip, argv[1], sizeof(auth_server_ip));
```
Let's not get into trouble by using strncpy or strncat. They may not null terminate the string. snprintf() may be a better choice here.

aravindprasads replied:
Hi @nmav 

Thanks for the review. Addressed your comments, used snprintf instead of string APIs. 
Kindly review the changes and let me know your views.

nmav commented:
I have simplified the patch a bit in https://github.com/radcli/radcli/pull/59
does it work ok for you?

aravindprasads commented:
> I have simplified the patch a bit in #59
> does it work ok for you?

Hi  @nmav 

Thanks for the patch. Yes, the code looks clean now. I have incorporated them in this pull request.  
Kindly review and let me know your views.
